crates/wit-parser/src/ast.rs

```diff
@@ -192,6 +200,56 @@ impl<'a> Interface<'a> {
 pub enum InterfaceItem<'a> {
     TypeDef(TypeDef<'a>),
     Value(Value<'a>),
+    Use(Use<'a>),
+}
+
+pub struct Use<'a> {
+    pub names: Option<Vec<UseName<'a>>>,
```

Comment:
I think it's ok to remove the `*` syntax since I don't think we really need that much any more and can always add that in later. For now I'm leaving that out of https://github.com/WebAssembly/component-model/pull/141

Reply:
Should i also make a pass through the tests and delete any `use * ...` related ones.